lib/web_ui/lib/src/engine/canvaskit/skia_object_cache.dart

```diff
@@ -303,6 +318,11 @@ class SkiaObjectBox<R extends StackTraceDebugger, T extends Object> extends Skia
 
   void _initialize(R debugReferrer, T initialValue) {
     _update(initialValue);
+    if (Instrumentation.enabled) {
+      Instrumentation.instance.incrementCounter(
+        '${_skDeletable?.constructor.name} created',
```

Comment:
Could we get 'null created' and 'null deleted'? How should we make sense of those?

Reply:
I have not seen any nulls yet. I'm not sure how to make sense of that situation. I think let's wait until we get a null and find a solution then.